src-java/rule-manager/rule-manager-implementation/src/main/java/org/openkilda/rulemanager/RuleManagerImpl.java

```diff
@@ -198,6 +198,8 @@ public RuleManagerImpl(RuleManagerConfig config) {
 
         generators.add(flowRulesFactory.getIngressRuleGenerator(
                 flowPath, flow, encapsulation, overlappingIngressAdapters));
+        generators.add(flowRulesFactory.getInputLldpRuleGenerator(flowPath, flow, overlappingIngressAdapters));
+        generators.add(flowRulesFactory.getInputArpRuleGenerator(flowPath, flow, overlappingIngressAdapters));
         // todo: add arp, lldp, flow loop, flow mirror, etc
```

Comment:
please remove arp and lldp from TODO list

Reply:
fixed